--- deep_disfluency/corpus/tree_pos_map.py
+++ deep_disfluency/corpus/tree_pos_map.py
@@ -409,7 +409,7 @@
                 print treemap

                 if not self.errorlog == None:

                     self.errorlog.write(warning)

-                raw_input()

+                input()

                 return None

             return treemap

         except:

@@ -552,14 +552,14 @@
                     print "can't get POS from index: POSMap.get_POS(utt)"

                     print utt.swda_filename

                     print utt.transcript_index

-                    raw_input()

+                    input()

             else:

                 POS.append("null")

         if not len(POS) == len(utt.text_words()):

             print "uneven lengths: POSMap.get_POS mytree.py"

             print utt.swda_filename

             print utt.transcript_index

-            raw_input()

+            input()

         return POS

     

 class POSMapCorpus(dict):

@@ -589,7 +589,7 @@
                 print sorted(self.keys())

                 if not self.errorlog == None:

                     self.errorlog.write(warning)

-                raw_input()

+                input()

                 return None

             return posmap

         else:

@@ -602,7 +602,7 @@
                 print "map key", map_key

                 if not self.errorlog == None:

                     self.errorlog.write(warning)

-                raw_input()

+                input()

             return None

       

     def append(self, transFilename, uttTransNumber, wordPOSMap):
