--- deep_disfluency/language_model/util.py
+++ deep_disfluency/language_model/util.py
@@ -24,8 +24,8 @@
         """This function counts the ngrams in tokens (a list of things) 

         for each order between 1 and max_order"""

         l = len(tokens)

-        for i in xrange(l):

-            for d in xrange(1,max_order+1):

+        for i in range(l):

+            for d in range(1,max_order+1):

                 ngram_map = ngram_maps[d]

                 j=i+d

                 if not j > l:

@@ -291,7 +291,7 @@
     """

     res = [x] * n

     for l in listOfLists:

-        for i in xrange(n):

+        for i in range(n):

             res[i] = f(res[i],l[i])

     return res

 

@@ -308,7 +308,7 @@
     for lines in itertools.izip_longest(*[file]*n_processes):

         pool.map(f,filter(lambda x : not x is None,lines))

         if not timer is None:

-            for i in xrange(n_processes):

+            for i in range(n_processes):

                 timer.advance()

 

 def concurrent_process_file(f,file,pool=None):

@@ -359,8 +359,8 @@
         n_lines += 1

         tokens = tokenize_sentence(line)

         l = len(tokens)

-        for i in xrange(l):

-            for d in xrange(1,max_order+1):

+        for i in range(l):

+            for d in range(1,max_order+1):

                 j=i+d

                 if not j > l:

                     ngrams.add(glue_tokens(tokens[i:j],d))
